scripts/add_existing_baseyear.py

```diff
@@ -294,7 +314,7 @@ def add_heating_capacities_installed_before_baseyear(n, baseyear, grouping_years
     # https://ec.europa.eu/energy/studies/mapping-and-analyses-current-and-future-2020-2030-heatingcooling-fuel-deployment_en?redir=1
     # file: "WP2_DataAnnex_1_BuildingTechs_ForPublication_201603.xls" -> "existing_heating_raw.csv".
     # TODO start from original file
-
+    grouping_years = [1980, 1985, 1990, 1995, 2000, 2005, 2010, 2015, 2019]
```

Comment:
Does it make sense to remove the configurability here? If necessary, we could add separate grouping years for power and heat, and place warnings that they can’t extend to the first planning horizon.

Reply:
ok I changed it now that there is one `grouping_year_power` and one` grouping_year_heat` option in the `config.yaml`